### PR TITLE
[NEW] Custom message popups

### PR DESCRIPTION
**Pull request by Hudell**
Adds support to custom messagePopup configurations

### CODE REVIEW

File: app/ui-message/client/popup/messagePopupConfig.html [hunk 1]
```diff
@@ -5,5 +5,9 @@
 		{{> messagePopup popupSlashCommandsConfig}}
 		{{> messagePopup popupEmojiConfig}}
 		{{> messagePopup popupReactionEmojiConfig}}
+
```
Comment (ggazzo) [inline app/ui-message/client/popup/messagePopupConfig.html]: why not migrate all messagePopup to the same behavior ?

Comment (tassoevan) [reply app/ui-message/client/popup/messagePopupConfig.html]: @ggazzo I believe it's more readable that way.

**Comment by antkaz** [comment]
Hello. How I can use this?

**Comment by saurabhshahare** [comment]
> 
> 
> Hello. How I can use this?

Anything on this?

**Pull request merged**